packages/to-dts/lib/cli.js

```diff
@@ -27,27 +27,34 @@ const toDts = {
         description: 'Create const of generated type to export',
         type: 'string',
       },
-      output: {
+      'output.file': {
```

Comment:
Harmonize configuration with fromJsdoc. Down the road this permits adding additional configs in the `output` object.